build/build-bundle.js

```diff
@@ -75,7 +75,7 @@ const banner = `
  * @param {{minify: boolean}=} opts
  * @return {Promise<void>}
  */
-async function buildBundle(entryPath, distPath, opts = {minify: true}) {
+async function buildLrBundle(entryPath, distPath, opts = {minify: true}) {
```

Comment:
This function is used by many things via the CLI. I think you need to add to opts a `format` that defaults to iife.

Reply:
yupppppppp okay i see it now.
since the use wasn't picked up by TS i missed it.
roger that, i'll make it extensible.

Reply:
that's fair, took me a second to grok why this wasn't ok b/c of that too

Reply:
since this CL is getting a lil bit more involved...  i could also switch devtools to adopt the UMD pattern as well..  wdyt?

it'd mean we get to drop all of these in the devtools-entry: 
```
// @ts-expect-error
  self.X = X;
```

Reply:
sgtm. You'll probably need to disable the devtool CI check first? Might be weird given you can't roll latest to CDT just yet. Maybe you'd just need to cherry pick into branch-9 and roll from there to get CI to pass.